modules/ztp-single-node-clusters.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-single-node-clusters_{context}"]
+
+= Single node clusters
+
+You can use ZTP to deploy single node clusters to run distributed units (DUs) on small hardware footprints at disconnected far edge sites.
+A single node cluster runs {product-title} Container Platform on top of one bare-metal machine, hence the single node.
+Single edge node servers contain a single node with supervisor functions and worker functions on the same host that are deployed at low
+bandwidth or disconnected edge sites.
+
+Hosts are partitioned to isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved set of CPUs.
+This is useful for resource-constrained environments, such as single-node production deployments, where you want to reserve most of the
+CPU resources for user workloads and configure {product-title} to run on a fixed number of CPUs within the host.
+
+Single node clusters hosting a DU application is divided into the following configuration categories:
+
+* Common - Values are the same for all SNO sites managed by a hub cluster.
+* Pools of sites - Common across a pool of sites where a pool size can be 1 to _n_.
+* Site specific - Likely specific to a site with no overlap with other sites, for example, a vlan.
```

Comment:
Might just be me, but I can't tell what these configuration categories apply to.  Is it the DU application? The node?

s/SNO/single node cluster/

Reply:
Clarified here: "A single node cluster hosting a DU application _on a node_...